src/components/Warehouse/WarehouseDetailsContainer.js

```diff
@@ -0,0 +1,85 @@
+import editIcon from "../../assets/icons/edit-white-24px.svg";
+import "./WarehouseDetailsContainer.scss";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import BackIcon from "../../assets/icons/arrow_back-24px.svg";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
```

Comment:
This should be combined to be more efficient.

`import { useParams, useNavigate } from "react-router-dom";
`